Configuration/Settings.yaml

```diff
@@ -4,6 +4,11 @@ Flowpack:
       enableLiveAsyncIndexing: true
       # Change size of single batch jobs via this setting
       batchSize: 500
+
+      # Number of failed jobs accepted to switch the alias to the new index
+      # -1 means, accept any number of failures
+      acceptedFailedJobs: -1
```

Comment:
Maybe we can have a more strict default as 0 ? No ?

Reply:
I would agree. -1 mimics the previous bahavior, so I thought it would be less breaking. But with the dependency change to the jobqueue we are kind of breaking anyway.